package.json

```diff
@@ -217,6 +218,7 @@
     "inspect-brk-docker": "node --experimental-modules  ./.reaction/scripts/run.mjs --inspect-brk=0.0.0.0:9229",
     "lint": "eslint .",
     "lint:errors": "eslint . --quiet",
+    "lint:gql": "graphql-schema-linter **/schemas/*.graphql",
```

Comment:
Why are we globbing on `/schemas`? There's several `.graphql` files that don't match this glob I think.

Reply:
I couldn't really find a better way to do it. We need a glob that omits all `.graphql` files that are client/query files because this doesn't validate those. By convention our server schemas are almost always in a `schemas` folder so I landed on this. I'm open to any suggestions.